daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -804,8 +815,10 @@ case class ParserTestCase(ptc: NodeSeq, parentArg: DFDLTestSuite)
       case (Some(_), None) => {
         compileResult.left.foreach { diags => throw TDMLException(diags, implString) }
         compileResult.right.foreach {
-          case (_, processor) =>
-            runParseExpectSuccess(processor, dataToParse, nBits, optExpectedWarnings, optExpectedValidationErrors, validationMode, roundTrip, implString)
+          case (_, proc) => {
+            processor = proc
+            runParseExpectSuccess(dataToParse, nBits, optExpectedWarnings, optExpectedValidationErrors, validationMode, roundTrip, implString)
+        }
```

Comment:
indentation wrong for this curly brace.

Reply:
I have been hesitant to turn on auto-formatting on IDEA thusfar. At some point I may try it. Right now I have to decide to use it manually.